### PR TITLE
Bug 26391: Define how to set a proxy using capabilities

### PR DESCRIPTION
**Pull request by AutomatedTester**

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -893,6 +893,68 @@
   </p>
 </section>
 <section>
+  <p>To <dfn>set the proxy</fdn> to <var>proxy</var>, the <a>remote end steps</a> must run the following:
```
Comment (andreastt) [inline webdriver-spec.html]: s/fdn/dfn/

File: webdriver-spec.html [hunk 2]
```diff
@@ -893,6 +893,68 @@
...
+    <ol>
+      <li><p>Let <var>proxy</var> be the JSON Object passed in as an argument</p></li>
```
Comment (andreastt) [inline webdriver-spec.html]: Wouldn’t this be the named _proxy_ argument passed in?

File: webdriver-spec.html [hunk 3]
```diff
@@ -893,6 +893,68 @@
...
+      <li><p>Let <var>proxy type</var> be the result of <a>getting a property</a> named
+        <code>proxyType</code> from <var>proxy</var>. If <var>proxy type</var> is undefined return
```
Comment (andreastt) [inline webdriver-spec.html]: `"<code>proxyType</code>"`, with quote marks

File: webdriver-spec.html [hunk 4]
```diff
@@ -893,6 +893,68 @@
...
+          an <a>error</a>. If <var>proxy type</var> is not a string with the one of the following
```
Comment (andreastt) [inline webdriver-spec.html]: Can we make the "If proxy type is undefined" part a separate step?

File: webdriver-spec.html [hunk 5]
```diff
@@ -893,6 +893,68 @@
...
+          values , return an <a>error</a>.
```
Comment (andreastt) [inline webdriver-spec.html]: Can we make these individual steps?

File: webdriver-spec.html [hunk 6]
```diff
@@ -893,6 +893,68 @@
...
+         </p>
+      </li>
+
+      <li><p>Match the value of <var>proxyType</var> against the following
+      options:
+          <dl>
+            <dt>The string <code>"pac"</code>:</dt>
+            <dd><p>If the implementation supports
+              <a href='https://web.archive.org/web/20070602031929/http://wp.netscape.com/eng/mozilla/2.0/relnotes/demo/proxy-live.html'>proxy autoconfiguration</a>,
```
Comment (andreastt) [inline webdriver-spec.html]: Is this the best reference we can get?

Comment (AutomatedTester) [inline webdriver-spec.html]: yes, @jgraham suggested it

Comment (andreastt) [inline webdriver-spec.html]: That’s terrifying.

File: webdriver-spec.html (hunk 5 above)
Comment (AutomatedTester) [inline webdriver-spec.html]: I have removed them now. The Match should take care of them

File: webdriver-spec.html (hunk 2 above)
Comment (AutomatedTester) [inline webdriver-spec.html]: probably. Removed the line

File: webdriver-spec.html (hunk 1 above)
Comment (AutomatedTester) [inline webdriver-spec.html]: sorted

**Comment by andreastt** [comment]
Looks good enough for me.

**Comment by AutomatedTester** [comment]
landed in 5b8f75d53ba2c158c5afe44641f03e1784e069e5

**Pull request closed without merging**